release.nix

```diff
@@ -296,26 +298,40 @@ let
   makeDeb_x86_64 = makeDeb "x86_64-linux";
 
   makeDeb =
-    system: diskImageFun: extraPackages: extraDebPackages:
+    system: diskImageFun: extraPackages: extraDebPackages: initSystem:
 
     with import <nixpkgs> { inherit system; };
 
     releaseTools.debBuild {
       name = "nix-deb";
       src = jobs.tarball;
+
       diskImage = (diskImageFun vmTools.diskImageFuns)
         { extraPackages =
-            [ "libdbd-sqlite3-perl" "libsqlite3-dev" "libbz2-dev" "libwww-curl-perl" "libcurl-dev" "libcurl3-nss" "libssl-dev" "liblzma-dev" ]
-            ++ extraPackages; };
+            [ "build-essential" "debhelper" "bison" "flex"
+              "libdbd-sqlite3-perl" "libsqlite3-dev" "libbz2-dev" "libwww-curl-perl" "libcurl-dev" "libcurl4-openssl-dev" "libssl-dev" "liblzma-dev"
+            ] ++ extraPackages; };
       memSize = 1024;
       meta.schedulingPriority = 50;
-      postInstall = "make installcheck";
-      configureFlags = "--sysconfdir=/etc";
+
+      phases = [ "unpackPhase" "installPhase" ];
+
+      postInstall = ''
+        make installcheck
+      '';
+
+      installPhase = ''
```

Comment:
Just a nitpick, but this seems like this could be split into patch, install, and build phases. Can we maybe even move this logic back into "debBuild"?